src/server/api/controllers/quiz-results.controller.js

```diff
@@ -0,0 +1,35 @@
+const knex = require('../../config/db');
+
+const getQuizResults = async (answerId, userId) => {
+  try {
+    let filteredResults;
+    if (answerId !== undefined && userId !== undefined) {
```

Comment:
This works fine but you could do this like this as well
```suggestion
    if (answerId && userId) {
```